docs/DEVELOPMENT.md

```diff
@@ -265,25 +299,116 @@ Finally, the datasource class will need to be included in the bootstrap file:
 require_once 'class.usage_datasource.php';
 ```
 
-### <a name="abcd"></a> Resource types
+### Resource types
 
 When implementing `get_supported_resources` two types of resources can be returned.
 
-*TieredResource*
+#### TieredResource
 
 The tiered resource requires a name, an identifier and an array of units that can be selected by users.
 This type of resource will allow the user to define usage tiers that will be used for price calculation.
 
 When using this resource type, the `fetch` method will return amounts in a unit that can then be converted to the selected unit by the `convert_amount` method.
 
-*MarkupResource*
+#### MarkupResource
 
 The markup resource requires only a name and an identifier.
 This type of resource is used when the external system already calculates the price of usage. It will allow
 the user to select a markup, either as a percentage or a fixed amount, to be added to the price calculated by the external system.
 
 When using this resource type, the `fetch` method will return prices directly.
 
+## Development Tools
+
+### Plugin Logger
+Plugin Logger is available for use to track what PluginSystem hooks are being fired and how your plugins are reacting to them.
```

Comment:
Plugin Logger tracks what PluginSystem hooks are being fired and how your plugins are reacting to them.